components/clarity-repl/src/repl/datastore.rs

```diff
@@ -628,12 +634,15 @@ impl Datastore {
             current_epoch: StacksEpochId::Epoch2_05,
             current_epoch_start_height: stacks_chain_height,
             constants,
+            remote_network_info: None,
+            client,
         }
     }
 
     fn new_with_remote_data(
         clarity_datastore: &ClarityDatastore,
         constants: StacksConstants,
+        client: HttpClient,
```

Comment:
Why pass the `client` and then use `clarity_datastore.client` anyway. Is the diff just deceiving me?

Reply:
`client `is used in the `Datastore` constructor, but it's not used to fetch the sortition on 669.

Reply:
I think that might be fine.

Reply:
Yes I think both are fine, it doesn't matter much which one we use they are basically the same thing.
In the past I used `clarity_datastore.client` because it was available there. But actually, now that I pass a client as the argument, i might as well just use it 👍 
I made the change @tippenein